sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyVaultKey.java

```diff
@@ -106,14 +106,19 @@ private void unpackKeyMaterial(Map<String, Object> key) {
         this.key = properties.createKeyMaterialFromJson(key);
     }
 
-    @JsonProperty(value = "kid")
-    private void unpackKid(String kid) {
-        properties.unpackId(kid);
```

Comment:
Validate that kid is not sent in a different location in V7.0 SDK .

Reply:
`kid` is only a part of the `JsonWebKey` object definition in the Swagger for service version 7.0, which is part of `KeyBundle` (mapped to the `KeyVaultKey` model) but not a direct member of it. It's deserialized in by `unpackKid()` in `unpackKeyMaterial()`.